js/components/controlBar.js

```diff
@@ -553,6 +557,31 @@ var ControlBar = createReactClass({
         </div>
       ),
 
+      /**
+       * This function returns the chromecast button definition based on if
+       * it's connected to a chromecast or not
+       * @private
+       * @return {Object} The button definition
+       */
+      chromecast: function() {
+        let castIcon = "chromecast-disconnected";
+        if (this.props.controller.state.chromecast.isConnected === true)
+        {
```

Comment:
Sorry, minor nitpick but can we move this bracket to the line above? This is in order to be consistent with the styling on the rest of the file